main.js

```diff
@@ -0,0 +1,56 @@
+function getTasks() {
+  const tasks = JSON.parse(localStorage.getItem('tasks'));
```

Comment:
- [ ] You can replicate this by completely deleting "tasks" from your local storage. When I load your app on the browser I get this error message. So when I add a book nothing happens because due to this error, a book is not added when no book is added no book can be loaded.

  ![ududid](https://user-images.githubusercontent.com/54848363/205923759-a82ff88e-346c-4c02-8904-1053c5d7bc80.jpg)

  You can fix this by initializing `tasks` to an empty array if it is `null`
  ```
  const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
  ```